src/hotspot/share/gc/shenandoah/shenandoahGlobalGeneration.cpp

```diff
@@ -78,13 +78,12 @@ size_t ShenandoahGlobalGeneration::available() const {
   return MIN2(available, ShenandoahHeap::heap()->free_set()->available());
 }
 
-size_t ShenandoahGlobalGeneration::soft_available() const {
-  size_t available = this->available();
-
-  // Make sure the code below treats available without the soft tail.
-  assert(max_capacity() >= ShenandoahHeap::heap()->soft_max_capacity(), "Max capacity must be greater than soft max capacity.");
-  size_t soft_tail = max_capacity() - ShenandoahHeap::heap()->soft_max_capacity();
-  return (available > soft_tail) ? (available - soft_tail) : 0;
+size_t ShenandoahGlobalGeneration::soft_available_exclude_evac_reserve() const {
```

Comment:
Two questions:
* How is this override different from the default implementation now?
* Should we not also take the minimum of this value and `free_set()->available()` as we do elsewhere?